libs/payments/cart/src/gql/cart.resolver.ts

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
+import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
+import { Cart } from '../../../../shared/db/mysql/account/src';
+import { InvoiceFactory } from '../lib/factories';
+import { CartManager } from '../lib/manager';
+import {
+  CartIdInput,
+  Cart as CartType,
+  SetupCartInput,
+  UpdateCartInput,
+} from './index';
+
+@Resolver((of: any) => CartType)
+export class CartResolver {
+  private cartManager: CartManager;
+  constructor(private log: MozLoggerService) {
+    this.cartManager = new CartManager(this.log);
+  }
+
+  @Query((returns) => CartType, { nullable: true })
+  public async cart(): Promise<CartType | null> {
+    // Query just for testing purposes
+    // TODO - Implement
+    const cart = await Cart.query().first();
+    if (!cart) {
+      return null;
+    }
+
+    return {
+      ...cart,
+      nextInvoice: InvoiceFactory(), // Temporary
+    };
+  }
+
+  @Mutation((returns) => CartType, { nullable: true })
+  public async setupCart(
+    @Args('input', { type: () => SetupCartInput })
+    input: SetupCartInput
+  ): Promise<CartType | null> {
+    return this.cartManager.setupCart({
+      ...input,
+      amount: 0, // Hardcoding to 0 for now. In future will be populated from Stripe request?
```

Comment:
Cart manager setupCart would handle calculating amount.

Reply:
Updated the SetupCart type to remove `amount`, and moved the temporary hard coded logic into `cartManager.setupCart`.